magma/lib/magma/actions/add_model.rb

```diff
@@ -23,15 +23,21 @@ def has_parent?
     end
 
     def create_model
-      Magma.instance.db[:models].insert(
-          project_name: @project_name,
-          model_name: @action_params[:model_name]
-      )
+      params = {
+        project_name: @project_name,
+        model_name: @action_params[:model_name]
+      }
 
-      project.load_model(
-          project_name: @project_name,
-          model_name: @action_params[:model_name]
-      )
+      # Here dictionary needs to be a JSON string, otherwise
+      #   Sequel will try to find a column for each dictionary key.
+      params[:dictionary] = JSON.generate(@action_params[:dictionary]) if @action_params[:dictionary]
+
+      Magma.instance.db[:models].insert(params)
```

Comment:
might set `params = { ... dictionary: @action_params[:dictionary] }` and then do like `Magma.instance.db[:models].insert(params.merge(dictionary: params[:dictionary].to_json))` to avoid the rewrite.